flexget/components/emby/api_emby.py

```diff
@@ -117,6 +117,19 @@ def update_using_map(target, field_map: dict, source_item, **kwargs):
 
                 break
 
+    @staticmethod
+    def strip_year(name: str) -> str:
+        new_name = name
+
+        if not isinstance(name, str):
+            return name
+
+        new_name = re.sub(r'^(.*) \(\d{4}\)(.*)', r'\1\2', new_name)
```

Comment:
Wouldn't it be cleaner to search and strip the year disregarding the wildcards before and after?
`new_name = re.sub(r' ?\(\d{4}\)', '', new_name)`